crates/symbolicli/src/main.rs

```diff
@@ -0,0 +1,655 @@
+use std::collections::HashMap;
+use std::fmt;
+use std::io::{Read, Seek, SeekFrom};
+use std::path::Path;
+use std::sync::Arc;
+
+use symbolic::common::split_path;
+use symbolicator_service::services::symbolication::SymbolicationActor;
+use symbolicator_service::types::{CompletedSymbolicationResponse, FrameTrust, Scope};
+use symbolicator_sources::{SentrySourceConfig, SourceConfig, SourceId};
+
+use anyhow::Context;
+use prettytable::format::consts::FORMAT_CLEAN;
+use prettytable::{cell, row, Row, Table};
+use reqwest::{header, Url};
+use tempfile::{NamedTempFile, TempPath};
+
+mod settings;
+
+#[tokio::main]
+async fn main() -> anyhow::Result<()> {
+    let settings::Settings {
+        event_id,
+        project,
+        org,
+        auth_token,
+        base_url,
+        symbolicator_config,
+        output_format,
+    } = settings::Settings::get()?;
+
+    tracing_subscriber::fmt::init();
+
+    let runtime = tokio::runtime::Handle::current();
+    let (symbolication, _objects) =
+        symbolicator_service::services::create_service(&symbolicator_config, runtime)
+            .await
+            .context("failed to start symbolication service")?;
+
+    let mut headers = header::HeaderMap::new();
+    headers.insert(
```

Comment:
We can use `append` here as we don't care about return value, but it's just a nitpick, feel free to ignore.